src/zac/conf/settings.py

```diff
@@ -13,10 +13,16 @@
 import os
 
 from django.urls import reverse_lazy
+from .local_example import *
```

Comment:
don't do this, it's an example file. Local overrides are imported at the bottom. You are supposed to copy `local_example.py` to `local.py` which is not kept in version control (with a reason).